pulseapi/entries/views.py

```diff
@@ -17,11 +17,16 @@
 from rest_framework.parsers import JSONParser
 
 from pulseapi.entries.models import Entry, ModerationState
-from pulseapi.entries.serializers import (
+from pulseapi.entries.serializers.base import (
     EntrySerializerWithV1Creators,
     EntrySerializerWithCreators,
     ModerationStateSerializer,
 )
+from .serializers.project import ProjectEntrySerializer
```

Comment:
if we switch to the module approach I mentioned below, we can simply do `from .serializers import ProjectEntrySerializer, NewsEntrySerializer, ...`